README.md

```diff
@@ -1,6 +1,4 @@
-The [FreeRTOS 202112.00](https://github.com/FreeRTOS/FreeRTOS/tree/202112.00) release
-adds the [Fleet Provisioning for AWS IoT](https://github.com/aws/Fleet-Provisioning-for-AWS-IoT-embedded-sdk/) library and the [Sigv4 library](https://github.com/aws/SigV4-for-AWS-IoT-embedded-sdk/). Additionally, the update includes an [example demonstrating the use of the Fleet Provisioning library](https://github.com/FreeRTOS/FreeRTOS/tree/release-candidate/FreeRTOS-Plus/Demo/AWS/Fleet_Provisioning_Windows_Simulator/Fleet_Provisioning_With_CSR_Demo), as well as a modification to the [HTTP S3 download demo](https://github.com/FreeRTOS/FreeRTOS/tree/release-candidate/FreeRTOS-Plus/Demo/coreHTTP_Windows_Simulator/HTTP_S3_Download) to demonstrate the Sigv4 library.
-This release also includes CBMC proofs for all public and private functions in the [OTA Update library](https://github.com/aws/ota-for-aws-iot-embedded-sdk). The version of mbed TLS used in corePKCS11 and in demos has been updated.
+The [FreeRTOS 202211.00](https://github.com/FreeRTOS/FreeRTOS/tree/202211.00) release updates FreeRTOS Kernel, FreeRTOS+TCP, coreMQTT, corePKCS11, coreHTTP, coreJSON, AWS IoT Over-the-air-Updates (OTA), AWS IoT Device Shadow, AWS IoT Jobs, AWS IoT Device Defender, Backoff Algorithm, AWS IoT Fleet Provisioning, coreSNTP, SigV4, and FreeRTOS Cellular Interface libraries to their [LTS 2.0](https://www.freertos.org/lts-libraries.html) versions. It also updates coreMQTT Agent to v1.2.0 to be compatible with coreMQTT v2.X.X, and updates MbedTLS to v3.2.1. This release also adds Visual Studio static library projects for the FreeRTOS Kernel, FreeRTOS+TCP, Logging, MbedTLS, coreHTTP, and corePKCS11. With the addition of the static library projects, all Visual Studio projects have been updated to use them. Additionally, all demos dependent on coreMQTT have been updated to work with coreMQTT v2.X.X.
```

Comment:
Instead of hyperlinking the LTS 2.0 changes to the freertos org, it might make sense to link to the LTS 2.0 CHANGELOG.md file i.e. https://github.com/FreeRTOS/FreeRTOS-LTS/blob/202210-LTS/CHANGELOG.md
Think it might be a better end-user experience for when trying to find out what's changed?